data/Templates/Ccda/ValueSet/ValueSet.json

```diff
@@ -15,6 +15,9 @@
       },
       "BAD": {
         "code": "old"
+      },
+      "__default__": {
```

Comment:
These default values should be consistent with the *.liquid templates in `ValueSet` folder. Let's add default values in the *.liquid templates as well.

Reply:
I found the updates in *.liquid templates, never mind about this comment :)